libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/RepeatSelect/RepeatExecuteOnSelect.tsx

```diff
@@ -11,6 +11,14 @@ import { messages } from "../../../../../locales";
 
 const DROPDOWN_WIDTH = 154;
 
+const getLocalizationKey = (id: string): { id: string } => {
+    if (id === REPEAT_EXECUTE_ON.DAY_OF_MONTH) {
+        return messages.scheduleDialogEmailRepeatsExecuteOn_dayOfMonth;
+    }
+
+    return messages.scheduleDialogEmailRepeatsExecuteOn_dayOfWeek;
```

Comment:
I would rather use switch and throw exception for unknown value. If you will introduce another value, the app would produce incorrect values without telling you.

Reply:
Changed it.